src/metpy/plots/plot_areas.py

```diff
@@ -0,0 +1,1014 @@
+                (-178.5, -170.5, -22.0, -15.0),
+                ccrs.PlateCarree()),
+    'tp': Areas('tp', 'SAO TOME-PRINCIPE',
+                (6.0, 7.6, 0.0, 2.0),
+                ccrs.PlateCarree()),
+    'ts': Areas('ts', 'TUNISIA',
+                (7.0, 13.0, 30.0, 38.0),
+                ccrs.PlateCarree()),
+    'tu': Areas('tu', 'TURKEY',
+                (25.0, 48.0, 34.1, 42.1),
+                ccrs.LambertConformal(standard_parallels=[30, 60], central_longitude=37)),
+    'tv': Areas('tv', 'TUVALU',
+                (176.0, 180.0, -11.0, -5.0),
+                ccrs.PlateCarree()),
+    'tw': Areas('tw', 'TAIWAN',
+                (120.0, 122.0, 21.9, 25.3),
+                ccrs.PlateCarree()),
+    'ug': Areas('ug', 'UGANDA',
+                (29.0, 35.0, -3.5, 5.5),
+                ccrs.PlateCarree()),
+    'uk': Areas('uk', 'UNITED KINGDOM',
+                (-11.0, 5.0, 49.0, 60.0),
+                ccrs.LambertConformal(standard_parallels=[30, 60], central_longitude=-3)),
+    'ur': Areas('ur', 'UKRAINE',
+                (20.0, 41.0, 45.0, 52.0),
+                ccrs.NorthPolarStereo(central_longitude=20)),
+    'uy': Areas('uy', 'URUGUAY',
+                (-60.0, -52.0, -35.5, -29.5),
+                ccrs.SouthPolarStereo(central_longitude=-56)),
+    'vanu': Areas('vanu', 'VANUATU',
+                  (167.0, 170.0, -21.0, -13.0),
+                  ccrs.PlateCarree()),
+    'vi': Areas('vi', 'VIRGIN ISLANDS',
+                (-65.5, -64.0, 16.6, 19.6),
+                ccrs.PlateCarree()),
+    'vk': Areas('vk', 'SLOVAKIA',
+                (13.8, 25.8, 46.75, 50.75),
+                ccrs.LambertConformal(standard_parallels=[30, 60], central_longitude=20)),
+    'vn': Areas('vn', 'VENEZUELA',
+                (-75.0, -60.0, -2.0, 14.0),
+                ccrs.PlateCarree()),
+    'vs': Areas('vs', 'VIETNAM',
+                (102.0, 110.0, 8.0, 24.0),
+                ccrs.PlateCarree()),
+    'wk': Areas('wk', 'WAKE ISLAND',
+                (166.1, 167.1, 18.8, 19.8),
+                ccrs.PlateCarree()),
+    'ye': Areas('ye', 'YEMEN',
+                (42.5, 54.1, 12.5, 19.1),
+                ccrs.PlateCarree()),
+    'yg': Areas('yg', 'YUGOSLAVIA',
+                (13.5, 24.6, 40.0, 47.0),
+                ccrs.LambertConformal(standard_parallels=[30, 60], central_longitude=19)),
+    'za': Areas('za', 'SOUTH AFRICA',
+                (16.0, 34.0, -36.0, -22.0),
+                ccrs.PlateCarree()),
+    'zb': Areas('zb', 'ZAMBIA',
+                (21.0, 35.0, -20.0, -7.0),
+                ccrs.PlateCarree()),
+    'zm': Areas('zm', 'WESTERN SAMOA',
+                (170.5, 173.5, -15.0, -13.0),
+                ccrs.PlateCarree()),
+    'zr': Areas('zr', 'ZAIRE',
+                (12.0, 31.6, -14.0, 6.0),
+                ccrs.PlateCarree()),
+    'zw': Areas('zw', 'ZIMBABWE',
+                (25.0, 34.0, -22.9, -15.5),
+                ccrs.PlateCarree())}
+
+with exporter:
+    areas = area_data
```

Comment:
Then generate the dict here and we avoid having the "name" in two spots:
```suggestion
with exporter:
    named_areas = {data.name: data for data in area_data}
```